### ISSUE TITLE
Intellisense Available Off-line

### ISSUE THREAD

**Issue by griffwampatuba** [nodejs/help#2050]
* v10.16.0-x64:
* Windows 7 Enterprise, SP 1:
* Scope = VS Code Editor Intellisense:

Hello,

I am new to learning Node.js.  I am using VS Code, and I have installed Node.js via the .msi application.

I notice that I do not have the following things available in my VS Code editor:

Intellisense for native Node.js modules
"Go to Definition" context menu (it's there in the context menu, but does not navigate anywhere)
"Go to Type Definition" (same thing, its' there in the context menu, but no results)

Is there a way to get the correct "Typing Definition Files" (or whatever is needed) offline?  I cannot reach out and grab the underlying information from the internet.

Thanks!

**Comment by tniessen** [comment]
Are you writing JavaScript or TypeScript code?

**Comment by gireeshpunathil** [comment]
ping @griffwampatuba

**Comment by griffwampatuba** [comment]
Thanks for the ping gireeshpunathil.

I'm writing JavaScript.  My work is offline, so as I've gone along I've had to learn and install additional things.  I have been able for the most part to get this functionality, after having slowly built up my environment.  I wish there was a way to do a full install from a single file for offline use.  There are lots of us that have to use Node in an isolated environment.
